### DISCUSSION TITLE
Global variables

### DISCUSSION THREAD

**Discussion by Kacem78** [elementor/elementor#21915] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

the global variables will help speed up the maintenance and creation of any pages/posts

### Describe the solution you'd like

just have a global table could be within the site setting where we hold all the global variables for the site, such as phone numbers, emails, adresse, names, etc , any thing that could be found on different place in the site, 
so if we want to modify let say just a phone number, we just then modify it on the global setting and that it
the global variable could be any where in the widgets, in the titles, in the texts , menus, etc
we can use it via a shortcode,  here is bellow in exemple for a phone number
let imagine in a title, rather then being hard coded such as "Call as now at number : +0123456789"
instead we can use : "Call as now at number : {{phone}}"  where phone is the global variable stored on the global site setting.

**Comment by vladninja** [comment]
Currently, I'm solving that by creating a dynamic tags and using in different places
